.github/workflows/clang-format-check.yml

```diff
@@ -1,5 +1,14 @@
 name: clang-format
-on: [push, pull_request]
+
+on:
+  push:
+    branches: [ develop ]
+    # Sequence of patterns matched against refs/tags
+    tags:
```

Comment:
I think maybe let's do this for all tags, not just this pattern.  That way if I specifically make tag 'TestV22', it will run.  If you can make that change, then I'd be happy to accept this.

Reply:
* or convince me otherwise...

Reply:
done